ndcube/utils/wcs.py

```diff
@@ -44,7 +44,9 @@ def inv(self):
     "PIXEL": "instr.pixel",
     "XPIXEL": "custom:instr.pixel.x",
     "YPIXEL": "custom:instr.pixel.y",
-    "ZPIXEL": "custom:instr.pixel.z"
+    "ZPIXEL": "custom:instr.pixel.z",
+    "HECR": "pos.bodyrc.alt",
```

Comment:
```suggestion
    "HECR": "custom:pos.heliographic.distance",
```

Reply:
I think `"HECR": "pos.bodyrc.alt"` makes sense but have suggested a change for `"HECR"`